src/shrutiDJun20Assignments/MeetingRoom.java

```diff
@@ -0,0 +1,114 @@
+package shrutiDJun20Assignments;
+
+public class MeetingRoom {
+
+	boolean nalandaBookingStatus, takshashilaBookingStatus,
```

Comment:
Here this all variable should be static only, what If I will create new object of class and check. the. status.